src/components/Reports/LocationSection.tsx

```diff
@@ -133,6 +185,51 @@ export default function LocationSection(props: LocationSectionProps): JSX.Elemen
     }
   };
 
+  const getLivePlants = useMemo(() => {
+    return latestObservation?.species.reduce((acc, sp) => (acc = acc + sp.permanentLive), 0);
+  }, [latestObservation]);
+
+  const getDeadPlants = useMemo(() => {
+    return latestObservation?.species.reduce((acc, sp) => (acc = acc + sp.cumulativeDead), 0);
+  }, [latestObservation]);
+
+  const getNumberOfPlots = useMemo(() => {
+    return latestObservation?.plantingZones.flatMap((pz) =>
+      pz.plantingSubzones.flatMap((subzone) => subzone.monitoringPlots)
+    ).length;
+  }, [latestObservation]);
+
+  const getMarkedAsComplete = useMemo(() => {
+    if (plantingSite) {
+      const totalArea = plantingSite.areaHa ?? 0;
+      const totalPlantedArea =
+        plantingSite?.plantingZones
+          ?.flatMap((zone) => zone.plantingSubzones)
+          ?.reduce((prev, curr) => (curr.plantingCompleted ? +curr.areaHa + prev : prev), 0) ?? 0;
+      const percentagePlanted = totalArea > 0 ? Math.round((totalPlantedArea / totalArea) * 100) : 0;
+      return `${percentagePlanted}%`;
+    }
+    return '0%';
+  }, [plantingSite]);
+
+  const getPlantingDensityForZones = useMemo(() => {
+    if (plantingSite && plantingDensity) {
+      const zoneNameWithDensities = plantingSite.plantingZones?.map(
+        (zone) => `${zone.name}: ${plantingDensity[zone.name]}`
+      );
+      return (
+        (
+          <Box>
+            {zoneNameWithDensities?.map((zd, index) => (
+              <Box key={`zone-${index}`}>{zd}</Box>
+            ))}
+          </Box>
+        ) || ''
+      );
+    }
+    return '';
+  }, [plantingSite, plantingDensity]);
```

Comment:
Thanks for the useMemo change 🙌 , one last suggestion, rename these by removing `get` prefix since they are no longer functions. Should be good to merge after that.